### PR TITLE
Codegen for openapi 4d4a107

### PR DESCRIPTION
**Pull request by richardm-stripe**
* Add support for `mandate` on `Charge`.
* Add support for `reference` on `SourceTransaction`.
* Add support for `person` on `TokenCreateParams
* Add support for new event types `payment_intent.canceled` and `setup_intent.canceled`
* Add support for `metadata` on `AuthorizationApproveParams` and `AuthorizationDeclineParams`
* Add `setMetadata` method on `AuthorizationUpdateParams`
* Rename "chidrens_and_infants_wear_stores" to "childrens_and_infants_wear_stores". This is technically a breaking change, but we've chosen to release this as a minor version as the previous name was virtually unused.

### CODE REVIEW

**Comment by ob-stripe** [comment]
Tagged as WIP to avoid merging while we fix the OpenAPI spec.

**Comment by richardm-stripe** [comment]
Changes that should not be in here:
-CardholderCreateParams.authorizationControls
-CardholderUpdateParams.authorizationControls
-CardCreateParams.authorizationControls
-CardUpdateParams.authorizationControls

Changes that belong:
+TokenCreateParams.person

+CardUpdateParams.setMetadata(EmptyParam)
+CardUpdateParams.setMetadata(Map<String, String>)
+TransactionUpdateParams.setMetadata(EmptyParam)
+TransactionUpdateParams.setMetadata(Map<String, String>)
+AuthorizationApproveParams.setMetadata(EmptyParam)
+AuthorizationApproveParams.setMetadata(Map<String, String>)
+AuthorizationDeclineParams.setMetadata(EmptyParam)
+AuthorizationDeclineParams.setMetadata(Map<String, String>)
+AuthorizationUpdateParams.setMetadata(EmptyParam)
+AuthorizationUpdateParams.setMetadata(Map<String, String>)

**Comment by ob-stripe** [comment]
LGTM, but assigning to @remi-stripe for another pair of 👀

File: src/main/java/com/stripe/model/Charge.java [hunk 1]
```diff
@@ -1675,6 +1675,10 @@ public void setRuleObject(Rule expandableObject) {
       /** Last four characters of the IBAN. */
       @SerializedName("last4")
       String last4;
+
+      /** ID of the mandate used to make this payment. */
+      @SerializedName("mandate")
+      String mandate;
```
Comment (remi-stripe) [inline src/main/java/com/stripe/model/Charge.java]: This in theory should be gated but I won't fight over this

File: src/main/java/com/stripe/param/issuing/CardCreateParams.java [hunk 2]
```diff
@@ -779,12 +779,12 @@ public Builder setInterval(Interval interval) {
...
 
-        @SerializedName("chidrens_and_infants_wear_stores")
-        CHIDRENS_AND_INFANTS_WEAR_STORES("chidrens_and_infants_wear_stores"),
-
         @SerializedName("child_care_services")
         CHILD_CARE_SERVICES("child_care_services"),
 
+        @SerializedName("childrens_and_infants_wear_stores")
+        CHILDRENS_AND_INFANTS_WEAR_STORES("childrens_and_infants_wear_stores"),
+
```
Comment (remi-stripe) [inline src/main/java/com/stripe/param/issuing/CardCreateParams.java]: MAke sure to mention this as a "breaking" change

**Pull request merged**